Guide/tailwindcss.markdown

```diff
@@ -56,85 +54,120 @@ mkdir tailwind
+      require('cssnano')({ preset: 'default' }),
+    ],
+  };
+```
+
+We also need a CSS entry point for Tailwind. Create a new file at `tailwind/app.pcss`. Note that `pcss` is the extension for PostCSS.
 
 ```css
-@tailwind base;
+@import "tailwindcss/base";
+@import "tailwindcss/components";
 
-@tailwind components;
+// Import any other files with your custom CSS.
+@import "components.pcss";
 
-.btn {
-    @apply px-4 py-2 bg-blue-600 text-white rounded;
-}
+// Must be the last import.
+@import "tailwindcss/utilities"; 
+```
 
-form > div {
-    @apply mb-4;
-}
+Finally, let's add a custom `pcss` file at `tailwind/components.pcss` demonstrating some of the customization we could do on our site:
 
-form input {
-    @apply shadow;
-    @apply appearance-none;
-    @apply border;
-    @apply rounded;
-    @apply w-full;
-    @apply py-2;
-    @apply px-3;
-    @apply text-gray-700;
-    @apply leading-tight;
+```css
+@layer base {
+    h1 {
+      @apply text-3xl;
+    }
+    h2 {
+      @apply text-2xl;
+    }
+
+    h3 {
+        @apply text-xl;
+    }
 }
 
-form label {
-    @apply block;
-    @apply text-gray-700;
-    @apply text-sm;
-    @apply font-bold;
-    @apply mb-2;
+
+
+// Add you CSS or import other post-css files.
+@layer components {
+    .btn {
+        @apply px-4 py-2 bg-blue-600 text-white rounded;
+    }
 }
 
-@tailwind utilities;
+// You can also nest CSS like in SCSS
+/*
+.foo {
+    .bar {
+        @apply text-white;
+    }
+}
+*/ 
 ```
 
+
 ### Adding the build step
 
 We need to add a new build command for starting a tailwind build process to our `Makefile`. For that append this to the `Makefile` in your project:
 
 ```makefile
 tailwind-dev:
-    ls tailwind/*.css|NODE_ENV=development entr npx tailwindcss build -i tailwind/app.css -o static/app.css -c tailwind/tailwind.config.js
+			cd tailwind && npx postcss ./app.pcss --output=../static/app.css --watch --verbose
 ```
 
 **Make requires tab characters instead of 4 spaces in the second line. Make sure you're using a tab character when pasting this into the file**
 
 This defines a new command `make tailwind-dev` that runs `npx tailwindcss build` whenever a CSS file inside the `tailwind/` directory changes. The CSS output will be placed at `static/app.css` (the standard main CSS file of IHP apps). It will use the tailwind configuration at `tailwind/tailwind.config.js`.
 
-For production builds we also need a new make target:
+For production builds, we also need a new make target:
 
 ```makefile
 static/app.css:
-    NODE_ENV=production npm ci
-    NODE_ENV=production npx tailwindcss build -i tailwind/app.css -o static/app.css -c tailwind/tailwind.config.js --minify
+	cd tailwind && npx postcss ./app.pcss --output=../static/app.css --verbose
 ```
 
 **Make requires tab characters instead of 4 spaces in the second line. Make sure you're using a tab character when pasting this into the file**
 
+You can now execute it with `make -B static/app.css`
```

Comment:
without the `-B` I've noticed it didn't always compile - and ended with `make 'static/app.css' is up to date` message